### PR TITLE
fix(#2552): unable to use from CDN

### PR DESCRIPTION
**Pull request by illiteratewriter**
- [x] Bug fix 
- [ ] New feature 
- [ ] Chore 
- [ ] Breaking change 
- [x] There is an open issue which this change addresses
- [x] I have read the **[CONTRIBUTING](./CONTRIBUTING.md)** document.
- [x] My commits follow the [Git Commit Guidelines](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#commits)
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly.
- [ ] My change requires a change to [Typescript typings](./types/lib).
  - [ ] I have updated the typings accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "cover": "npm test -- --coverage",
     "start": "start-storybook -s ./static -p 8080",
     "build:docs": "build-storybook -s ./static -o ./build",
-    "build": "microbundle --generateTypes false --jsx React.createElement --compress false",
+    "build": "microbundle --generateTypes false --jsx React.createElement --compress false --globals react=React,react-dom=ReactDOM,prop-types=PropTypes,react-transition-group=ReactTransitionGroup,react-popper=ReactPopper,classnames=classNames",
```
Comment (davidacevedo) [inline package.json:23]: Would this affect importing the library without those globals being defined?

Comment (illiteratewriter) [reply package.json:23]: I didn't understand the question, but the UMD build was not working without these globals. If you remove `react=React` from the above, then it'll throw `React__default.default is undefined`. In [this issue](https://github.com/reactstrap/reactstrap/issues/2552), it throws error for `propTypes`.

**Comment by davidacevedo** [comment]
I'll try testing this a bit more later to see if I can validate my assuptions. It does look good otherwise

**Comment by illiteratewriter** [comment]
Wouldn't be needed as we are moving away from microbundle

**Pull request closed without merging**